src/main/java/mate/academy/intro/model/User.java

```diff
@@ -26,4 +36,28 @@ public class User {
     @Column(nullable = false)
     private String lastName;
     private String shippingAddress;
+    @ManyToMany(fetch = FetchType.EAGER)

```

Comment:
```suggestion
    @ManyToMany
```
Use join fetch or EntityGraph